dGame/dComponents/TriggerComponent.cpp

```diff
@@ -0,0 +1,249 @@
+#include "TriggerComponent.h"
+
+#include "dZoneManager.h"
+#include "LUTriggers.h"
+
+#include "MissionComponent.h"
+#include "PhantomPhysicsComponent.h"
+
+#include "CDMissionTasksTable.h"
+
+
+TriggerComponent::TriggerComponent(Entity* parent, const std::string triggerInfo): Component(parent) {
+	m_Parent = parent;
+	m_Trigger = nullptr;
+
+	std::stringstream ss(triggerInfo);
+	std::vector<std::string> tokens;
+	std::string token;
+	while (std::getline(ss, token, ':')) {
+		tokens.push_back(token);
+	}
+
+	uint32_t sceneID = std::stoi(tokens[0]);
+	uint32_t triggerID = std::stoi(tokens[1]);
```

Comment:
For sanity TryParse might be good to have

Reply:
will do, just copy pasted what was there originally